pkg/dogstatsd/server.go

```diff
@@ -341,16 +342,47 @@ func (s *Server) Flush(waitForSerializer bool) {
 	s.aggregator.Flush(time.Now().Add(time.Second*10), true)
 }
 
+// dropCR drops a terminal \r from the data.
```

Comment:
I'm curious, is that really a thing? Having a `\r` ending a line instead of a `\n` sounds strange to me, was it while you were testing your code sending packets through the term? I'm not entirely sure this is worth having this if it is only for "manuel tests" but in the same way, it doesn't hurt that much, so that's fine. I think I'm just curious of how you met this because I may miss something.

Reply:
As you know the windows carriage return is `\r\n`, we don't know anything about how our customers might implement their dogstatsd clients so this felt like the safest thing to do; it does add a bit of overhead but hopefully nothing unbearable given the full cost of parsing.

Reply:
Makes total sense, thanks!